Midtrans/Transaction.php

```diff
@@ -90,6 +90,24 @@ public static function refund($id)
         );
     }
 
+    /**
+     * Transaction status can be updated into refund
+     * if the customer decides to cancel completed/settlement payment.
+     * The same refund id cannot be reused again.
+     * 
+     * @param string $id Order ID or transaction ID
+     * 
+     * @return mixed[]
+     */
+    public static function refundDirect($id, $params)
+    {
+        return ApiRequestor::post(
+            Config::getBaseUrl() . '/' . $id . '/refund/online/direct',
+            Config::$serverKey,
+            false
```

Comment:
Looks like the 3rd parameter should not be `false` but should be the `$params` (JSON equivalent of the payload body) direct refund require JSON body.

Like the charge fundction:
https://github.com/Midtrans/midtrans-php/blob/6d3b1f953e317b64b81a34b355ae72566ede4676/Midtrans/Snap.php#L79-L83

Please add the params